install/kubernetes/cilium/values.yaml.tmpl

```
@@ -387,12 +387,12 @@ cni:
   # This can be useful if you want to manage your CNI
   # configuration outside of a Kubernetes environment. This parameter is
   # mutually exclusive with the 'cni.configMap' parameter.
-  # readCniConf: /host/etc/cni/net.d/05-cilium.conf
+  readCniConf: /host/etc/cni/net.d/05-cilium.conf
```

to avoid file not found upon agent start

```suggestion
  # Example value: /host/etc/cni/net.d/05-cilium.conf
  readCniConf: ""
```